inc/colour.h

```diff
@@ -65,16 +65,16 @@ t_colour	ft_hadamard_colour(t_colour c1, t_colour c2);
  * @brief Converts a t_colour struct to an int32_t.
  *
  * @param colour	Colour to convert.
- * @return int32_t	Converted colour.
+ * @return uint32_t	Converted colour.
```

Comment:
Seems to be more inline with standard way of doing things. Has no effect on program since it's just representation of the underlying data > we don't care about the actual number but about the r,g,b channels